ShowMeTheRoapes/paracord-steam
# Dockerized the project …

…on unit test and uat test levels. Added an example UAT test that hits a live running local service. Updated readmes

## Proposed Changes

- Added docker files 
- Added a dockerized unittest runner
- Added dockerized local test environment 
- Added example UAT(integration) test

To test:
Follow the Readme instructions in the /server/ directory to see how to run the unittests and UAT test
